en/components/grid/grid.md

```diff
@@ -377,6 +377,189 @@ and in the template of the component:
+                <ng-template #nestedDataTemp igxCell let-cell="cell">
+                    <div class="employees-container">
+                            // For every employee record in the Employee field, render its Name, Title and HireDate value
+                        <div *ngFor="let employee of cell.value">
+                            <span><strong>Name:</strong> {{employee.Name}} </span>
+                            
+                            <span><strong>Title:</strong> {{employee.Title}} </span>
+                            
+                            <span><strong>Hire Date:</strong> {{employee.HireDate | date:'shortDate'}} </span>
+                            
+                        </div>
+                    </div>
+                </ng-template>
+...
+```
+Оnly with this template you will not be able to make **CRUD** operations, so we need an editor template.
+
+```html
+...
+                <ng-template igxCellEditor let-cell="cell">
+                    <div class="employees-container--edit">
+                        <div *ngFor="let employee of cell.value">
+                            <span>
+                                <strong>Name:</strong>
+                                <igx-input-group>
+                                    <input igxInput [(ngModel)]="employee.Name" />
+                                </igx-input-group>
+                            </span>
+                            <span>
+                            <strong>Title:</strong>
+                            <igx-input-group>
+                                <input igxInput [(ngModel)]="employee.Title" style="text-overflow:ellipsis" />
+                            </igx-input-group>
+                        </span>
+                            <span>
+                            <strong>Hire Date:</strong>
+                            <igx-date-picker [(ngModel)]="employee.HireDate" [label]="''"></igx-date-picker>
+                        </span>
+                        </div>
+                    </div>
+                </ng-template>
+</igx-column>
+...
+```
+
+And the result from this configuration is:
+
+<div class="sample-container loading" style="height:500px">
+    <iframe id="grid-nested-dataBind-iframe" data-src='{environment:demosBaseUrl}/grid/grid-nested-data-binding' width="100%" height="100%" seamless="" frameborder="0" class="lazyload"></iframe>
+</div>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="grid-nested-dataBind-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+
+#### Flat data
+
+The composite flat data binding approach is similar to the nested data, but instead of **cell value** we use the [`rowData`]({environment:angularApiUrl}/classes/igxrowcomponent.html#rowdata) property of the [IgxRowComponent]({environment:angularApiUrl}/classes/igxrowcomponent.html).
```

Comment:
```suggestion
The flat data binding approach is similar to the one that we already described above, but instead of **cell value** we are going to use the [`rowData`]({environment:angularApiUrl}/classes/igxrowcomponent.html#rowdata) property of the [IgxRowComponent]({environment:angularApiUrl}/classes/igxrowcomponent.html).
```